libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -179,6 +180,66 @@ bool IRGeneratorForStatements::visit(FunctionCall const& _functionCall)
 	return false;
 }
 
+// Preliminary implementation that doesn't take care of many things like
+// name collisions etc. Mainly used for test cases.
+bool IRGeneratorForStatements::visit(InlineAssembly const& _inlineAsm)
+{
+	auto const& loc = _inlineAsm.location();
+
+	solAssert(loc.start != 1 && loc.end != -1, "Expected source location");
```

Comment:
Why this? Looks like a rather generic test

Reply:
Ok I see now

Reply:
In fact we should assert ``loc.source`` (resp. ``!!loc.source`` ;-P) as well here.